scss/navbar.scss

```diff
@@ -0,0 +1,24 @@
+.navbar {
+  background-color: $white;
+  box-shadow: 7px 7px 5px $grey;
+  display: flex;
+  height: $nav-height;
+  position: fixed;
+  width: 100%;
+  z-index: 1;
+}
+
+.wolox-logo {
+  align-self: center;
```

Comment:
This is unnecessary, please delete it. You can assign `align-items: middle` in `navbar` class